timetable/views.py

```diff
@@ -4,6 +4,10 @@
 from datetime import date, datetime
 
 from io import TextIOWrapper
+from multiprocessing import context
```

Comment:
None of the following imports are actually used, moreover
secrets were introduced in Python 3.6, which is newer than current minimal version of 3.5 declared in the Readme. Keeping those imports would require some explanation and documentation update.